### ISSUE TITLE
Over-fitting problems trained on market datasets

### ISSUE THREAD

**Issue by RenYurui** [tengteng95/Pose-Transfer#39]
I have trained a model over the market dataset. (But I do not use the hyperparameters you provided.) I found that the performance of the model differs greatly between the training set and the test set.   
Have you ever encountered over-fitting problems when training over the market dataset?

Although there are 200000+ training pairs, only 1000+ persons included.
Meanwhile, as the backgrounds are different, the reconstruction loss will dominate, which I think will lead to over-fitting.

**Comment by jessemelpolio** [comment]
This is something very tricky. You're right. This over-fitting problem does exist. Generally, decreasing the training iterations will help as well. In my perspective, reconstruction loss helps to lead a good start of the training but may not dominate the training process in the near end. It can be seen from the value of this loss. In the beginning, the loss decreases quickly but after some time, it will fluctuate around a specific level, meaning it doesn't give effective supervision then. But this is just my personal thought and I didn't rigorously investigate it.
